_locales/en/messages.json

```diff
@@ -157,16 +157,20 @@
     "message": "Cancel",
     "description":"The button text which says 'Cancel'."
   },
-  "saveFilePrompt": {
-    "message": "$filename$ has been modified. Do you want to save it before closing?",
-    "description":"The text which prompts user if they want to save file before closing it.",
+  "saveFilePromptLine1": {
+    "message": "$filename$ has been modified.",
+    "description":"First line of the text which prompts user if they want to save file before closing it.",
```

Comment:
I would also add a description along the lines of "Tells user the file contents has changed" or something.

Reply:
Done. I feel like this is redundant though as it's just repeating what is in the message itself.